newsrevenuehub/github-pages-stripe-checkout
# adding dynamic input field

Plan? (if this draft/incomplete indicate what you intend to do and how)
This is the very first step in terms of the tasks you discussed, but I wanted to get a MVP working here to make sure I am heading in the direction you are looking for. Right now, this uses the script tag within the index.html file to allow a user to add in a custom donation amount that updates dynamically with the donate button.

My next step will be to split this script out into its own js file.

I have been working on changes to allow for the ability to agree to fees, pick frequency of recurring donations, and add those keys to the client reference id.

I will continue to commit working code as I go, making the necessary updates.

What's this PR do?
This PR tackles step one of dynamically updating the donate button based on a user-inputted donate amount.

Why are we doing this? How does it help us?
Users need to be able to have reign over how much they want to donate.

How should this be manually tested?
Check out the branch and try to add a donation amount in the input field. See if the button dynamically populates with this amount.

How should this change be communicated to end users?
N/A

Are there any smells or added technical debt to note?
NA

What are the relevant tickets?
newsrevenuehub/newsrevhub-stripe-prod#653

Are any database schema changes associated with this ticket? Have those changes been made to the test/staging instance?
No

Did you create any new field mappings to SF? Did you update the org's SF or the managed package to include those fields? Are the fields present on all relevant objects?
N/A

Have you done the following, if applicable:
(optional: add explanation between parentheses)

[ N/A] Added automated tests? ( )
[Yes ] Tested manually on mobile? (Things look a little strange style-wise on mobile but I am assuming for now we can ignore that? If not, happy to look into it )
[N/A ] Checked for security implications? ( )
[ N/A] Updated the documentation/wiki? ( )
